### ISSUE TITLE
Error on when trying to start build on production

### ISSUE THREAD

**Issue by schurig** [boldpoker/edeliver#42]
When I run `mix edeliver start production  --verbose`

it tells me

```
EDELIVER PROJECT WITH START COMMAND

-----> starting production servers

production node:

  user    : apps
  host    : example.com
  path    : /home/apps/test123-project-website

Pseudo-terminal will not be allocated because stdin is not a terminal.
Using /home/apps/test123-project-website/project/releases/0.0.1/project.sh

START DONE!
```

Any idea how I solve this `Pseudo-terminal will not be allocated because stdin is not a terminal.` issue?

**Comment by schurig** [comment]
Update: 

```sh
[ "${_node_index}" = "single_node" ] && local _terminal_option=-t || local _terminal_option=
  ssh $_terminal_option -o ConnectTimeout="$SSH_TIMEOUT" "${_user}@${_host}" "$_remote_job"
  local _status_code="$?"
```

In the *`strategies/erlang-node-execute`* file in **line 134** I have replaced the `ssh $_terminal_option -o ConnectTime...` thing with `ssh -T -o ConnectTime...`.
Before that the `$_terminal_option` was a lowercase `-t`.

For now it seems to not throw this error anymore and it also starts the server. But it doesn't feel right to patch it like that!

**Comment by bharendt** [comment]
The [ssh `-t` option](http://www.openbsd.org/cgi-bin/man.cgi/OpenBSD-current/man1/ssh.1) allocates a pseudo terminal which allows user interaction. This is enabled by edeliver if the __node command is executed on a single node__.
Scripts executed on the node can check whether there is a (pseudo) terminal and then maybe ask the user for input. E.g. the edeliver `migration` command could ask for confirmation etc.

The warning `Pseudo-terminal will not be allocated because stdin is not a terminal.` is printed only if some data are sent to stdin.

```sh
  $  ssh -t  user@host 'echo foo | read a; echo $a' 
  foo
  Connection to host closed.
  $ echo bar | ssh -t  user@host 'read a; echo $a' 
  Pseudo-terminal will not be allocated because stdin is not a terminal.
  bar
```

Actually nothing like that is done by edeliver when a node command is executed. But I figured out that
the mix tasks calling the actuall edeliver binary opens stdin when executed by `Mix.shell.cmd`.
So thanks for reporting this.

**Comment by mk** [comment]
@schurig: We've just released 1.1.2 to hex.pm which should fix the issues you've seen. Would be great if you could give it a try!
